browser/src/control/Control.LokDialog.js

```diff
@@ -660,6 +660,9 @@ L.Control.LokDialog = L.Control.extend({
 		// Override default minHeight, which can be too large for thin dialogs.
 		L.DomUtil.setStyle(dialogContainer, 'minHeight', height + 'px');
 
+		// Title bar may overflow due to range name. So we should have max width
+		L.DomUtil.setStyle(dialogContainer.previousSibling, 'maxWidth', width - 36 + 'px');
```

Comment:
36 is close button's size and margin. I think it doesn't worth to do complicated calculation here to find exact width . I'm open to suggestions if there is a simple way to get it.

Reply:
to which object points dialogContainer.previousSibling ?

Reply:
Titlebar